tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -349,6 +363,51 @@ describe(__filename, () => {
     expect(state.description).toEqual(secondCollection.description);
   });
 
+  it('renders editable collection add-ons', () => {
+    const extAddon1 = { ...fakeAddon, name: 'uBlock' };
+    const extAddon2 = { ...fakeAddon, name: 'AdBlockPlus' };
+
+    const collection = createInternalCollection({
+      detail: createFakeCollectionDetail(),
+      items: createFakeCollectionAddons({
+        addons: [extAddon1, extAddon2],
+      }).results,
+    });
+    const root = render({ collection });
+
+    const addons = root.find(EditableCollectionAddon);
+    expect(addons).toHaveLength(2);
+    expect(addons.at(0))
+      .toHaveProp('addon', createInternalAddon(extAddon1));
+    expect(addons.at(1))
+      .toHaveProp('addon', createInternalAddon(extAddon2));
+  });
+
+  it('handles searching for an add-on', () => {
+    const root = render();
+
+    const search = simulateAutoSearchCallback({
+      root, propName: 'onSearch',
+    });
+    search({ query: 'ad blocker' });
+    // TODO: test onSearch
+    // https://github.com/mozilla/addons-frontend/issues/4590
+  });
+
+  it('handles selecting an add-on', () => {
```

Comment:
I wonder if these tests should be set to be skipped for now? 🤷‍♂️

Reply:
We need them for test coverage to make sure each handler doesn't throw an error at the very least

Reply:
Ah, fair point.